great_expectations/rule_based_profiler/rule_based_profiler_result.py

```diff
@@ -70,7 +71,7 @@ def to_dict(self) -> dict:
             ],
             "citation": self.citation,
             "execution_time": self.execution_time,
-            "usage_statistics_handler": self.usage_statistics_handler.__class__.__name__,
+            "_usage_statistics_handler": self._usage_statistics_handler.__class__.__name__,
```

Comment:
Why were we ever including this in to_dict? there should be no reason to ever show this; please remove

Reply:
@donaldheppner The reason is "informational" for printing and/or troubleshooting purposes.  I am happy to make the names without the underscores.  We can of course remove "usage_statistics_handler" (with or without the underscore) from `to_dict()` entirely, but then we would lose knowing that the object actually had the usage stats handler, which, seems a small disadvantage compared to knowing its name.  Happy to discuss (easy change).  Thanks.